addons/interact_menu/ACE_Settings.hpp

```diff
@@ -1,26 +1,4 @@
 class ACE_Settings {
-    class GVAR(alwaysUseCursorSelfInteraction) {
-        value = 0;
-        typeName = "BOOL";
-        isClientSettable = 1;
-        category = CSTRING(Category_InteractionMenu);
-        displayName = CSTRING(AlwaysUseCursorSelfInteraction);
-    };
-    class GVAR(cursorKeepCentered) {
-        value = 0;
-        typeName = "BOOL";
-        isClientSettable = 1;
-        category = CSTRING(Category_InteractionMenu);
-        displayName = CSTRING(cursorKeepCentered);
-        description = CSTRING(cursorKeepCenteredDescription);
-    };
-    class GVAR(alwaysUseCursorInteraction) {
-        value = 0;
-        typeName = "BOOL";
-        isClientSettable = 1;
-        category = CSTRING(Category_InteractionMenu);
-        displayName = CSTRING(AlwaysUseCursorInteraction);
-    };
     class GVAR(useListMenu) {
```

Comment:
might aswell move the other settings to script and get rid of this config. But that might be a different PR

Reply:
I didn't think each setting needed to be separate since most people will probably leave them with the identical defaults.

Reply:
dedmen means just convert other settings to script version, not separating them

Reply:
Ah I see, yeah that can be done